### DISCUSSION TITLE
Is there a specific configuration/adapter spec for deno tests?

### DISCUSSION THREAD

**Discussion by feslima** [nvim-neotest/neotest#163] [Q&A]
I've tried to searching on how to configure neotest with deno but got no results.

Does anybody have any tips, recommendations on how to do so?

Thanks.

**Comment by rcarriga** [comment]
Not that I'm aware of but if someone wants to add support with an adapter, I'm happy to help! The neotest-jest and neotest-vitest repos might be a good place to start.

Otherwise vim-test supports deno so you can get some basic support for it but it definitely won't be as nice as first class support.

**Reply by feslima** [reply]
Thanks for the directions. I will see if I can come up with something that works.
